src/Backend/OnboardingInsuranceAPI/Areas/Insurance/CarInsuranceData.cs

```diff
@@ -0,0 +1,25 @@
+using System.Text.Json.Serialization;
+
+namespace OnboardingInsuranceAPI.Areas.Insurance;
+
+public record CarInsuranceData
+{
+    [JsonPropertyName("expDate")]
+    public string ExpDate { get; init; } //Expiration date
```

Comment:
Use full name in properties, since you are already using JsonPropertyName attribute.

Reply:
Then you don't need the comment.